src/software/simulation/er_force_simulator.cpp

```diff
@@ -425,3 +438,16 @@ void ErForceSimulator::resetCurrentTime()
 {
     current_time = Timestamp::fromSeconds(0);
 }
+
+std::map<RobotId, Vector> ErForceSimulator::getRobotIdToLocalVelocityMap(
+    const google::protobuf::RepeatedPtrField<world::SimRobot>& sim_robots)
+{
+    std::map<RobotId, Vector> robot_to_local_velocity;
+    for (const auto& sim_robot : sim_robots)
+    {
+        robot_to_local_velocity[sim_robot.id()] =
+            Vector(sim_robot.v_x(), sim_robot.v_y())
+                .rotate(Angle::fromRadians(sim_robot.angle()));
```

Comment:
just curious, does this rotation make it the velocity local or global?

Reply:
local, added a comment